### DISCUSSION TITLE
unable to logout

### DISCUSSION THREAD

**Discussion by smukk9** [coder/code-server#3267] [Q&A] [answered]
Hi , I am facing a couple of problems and appreciate if some can help me on this. 

My Enviroemnt :
- OS: Ubuntu
- using caddy to server code-server 

Here is my caddy file sample :
```
#for my static content
mydomain.com

root * /path/to/dir
file_server

# for code server
route /code/* {
  uri strip_prefix /code
  reverse_proxy 127.0.0.1:8080
}
```
My issues: 
1.  Doc here https://github.com/cdr/code-server/blob/main/docs/guide.md mention to use the domain/code/* but that does not work so i am using route directive which works for login and using the IDE on web. is that correct or is there a another way to server on subpath.

2.  Login works but logout is not working. When i hit the logout button it redirects to mydomain.com/login with 404.  This error is correct since i only have /code and /login tries look at my static files. how to fix the logout in this case ? 

Thanks in advance for any help.

**Comment by code-asher** [comment, marked as answer]
I'm not that familiar with Caddy but if it works then I say you're good to go!

The logout unfortunately is hard-coded to the root. There is an issue to fix it here: https://github.com/cdr/code-server/issues/2984
